data/stoa0169a/stoa002/stoa0169a.stoa002.opp-lat2.xml

```diff
@@ -42,18 +44,19 @@
             <sourceDesc>
                 <listBibl xml:lang="lat">
                     <biblStruct>
-                        <monogr>
-                            <editor>
+                        <monogr>                                                       
+                         <author>Juvencus, Caius Vettius Aquilinus</author>
+                          <title>Evangeliorum Libri Quattuor</title>
+                              <editor>
                                 <persName>
                                     <name xml:lang="lat">Iohannes Huemer</name>
-                                    <addName xml:lang="de">Johann Huemer</addName>
+                                    <addName xml:lang="deu">Johann Huemer</addName>
                                 </persName>
-                            </editor> 
-                          <author ref="http://data.perseus.org/catalog/urn:cite:perseus:cattg.1542.1">Iuvencius</author>  
-                          <title ref="http://data.perseus.org/catalog/urn:cts:latinLit:stoa0169a.stoa002.opp-lat2">Evangeliorum Libri Quattuor</title>
+                            </editor>
                             <imprint>
-                                <publisher>Österreichische Akademie der Wissenschaften (ÖAW)</publisher>
-                                <pubPlace>Wien, Österreich</pubPlace>
+                                <title>Corpus scriptorum ecclesiasticorum Latinorum</title>
```

Comment:
Title needs Lang attribute

Reply:
Hi @lcerrato I think I made all the necessary corrections. Thank you for reviewing.